python/spark_sklearn/tests/test_gapply.py

```diff
@@ -0,0 +1,211 @@
+
+from itertools import chain
+import pandas as pd
+import random
+import unittest
+
+from pyspark.conf import SparkConf
+from pyspark.sql import SparkSession
+from pyspark.sql.types import *
+from pyspark.sql import Row
+from pyspark.sql.tests import PythonOnlyPoint, PythonOnlyUDT, ExamplePoint, ExamplePointUDT
+
+from spark_sklearn.test_utils import fixtureReuseSparkSession
+from spark_sklearn.group_apply import gapply
+
+def _assert_frame_equal(actual, expected):
```

Comment:
rename to __assert_pdframe_equal so that it is clear which dataframe we are talking about